tomcat-main/src/main/resources/defaults/base-procedure-hit.xml

```diff
@@ -0,0 +1,86 @@
+<record id="hit" in-findedit="yes" type="record,procedure" cms-type="default" generate-services-schema="true">
+  <services-url>hits</services-url>
+  <services-tenant-singular>Hit</services-tenant-singular>
+  <services-tenant-plural>Hits</services-tenant-plural>
+  <web-url>hit</web-url>
+  <tab-url>hit-tab</tab-url>
+
+  <services-list-path>abstract-common-list/list-item</services-list-path>
+  <services-record-path>hits_common:http://collectionspace.org/services/hit,hits_common</services-record-path>
+  <services-record-path id="collectionspace_core">collectionspace_core:http://collectionspace.org/collectionspace_core/,collectionspace_core</services-record-path>
+
+  <include src="domain-procedure-hit.xml" strip-root="yes" />
+
+  <section id="coreInformation">
+    <include src="core-fields.xml" strip-root="yes" />
+  </section>
+
+  <section id="hitInformation">
+    <field id="hitNumber" mini="number,list" />
+    <field id="entryDate" datatype="date" />
+
+    <repeat id="hitDepositorGroupList/hitDepositorGroup">
+      <field id="depositor" mini="summary,list" autocomplete="true" />
+      <field id="depositorContact" autocomplete="true" />
+      <field id="depositorContactType" autocomplete="true" ui-type="enum" />
+      <field id="depositorNote" />
+    </repeat>
+
+    <repeat id="agreementGroupList/agreementGroup">
+      <field id="agreementStatus" ui-search="repeatable" autocomplete="true" ui-type="enum" />
+      <field id="agreementStatusDate" ui-search="repeatable" datatype="date" />
+      <field id="agreementStatusNote" ui-search="repeatable" />
+    </repeat>
+
+    <repeat id="entryMethods">
+      <field id="entryMethod" autocomplete="true" ui-type="enum" />
+    </repeat>
+
+    <repeat id="agreementRenewalDates">
+      <field id="agreementRenewalDate" datatype="date" ui-search="repeatable" />
+    </repeat>
+
+    <field id="entryReason" autocomplete="true" ui-type="enum" />
```

Comment:
This has `autocomplete="true" ui-type="enum"`, but it is bound to a static option list in the UI. Either these attributes should be removed, or it should be bound to a dynamic term list.